src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -38,30 +37,45 @@ public static void initialize(IntakeIO io) {
     }
 
     public Command setAngle(MutableMeasure<Angle> angle) {
-        inputs.angleSetPoint = angle;
-        return runOnce(() -> io.setAngle(angle));
+
+        return runOnce(
+                () -> {
+                    inputs.angleSetPoint = angle;
+                    io.setAngle(
+                            Units.Degrees.of(Math.IEEEremainder(angle.in(Units.Degrees), 180))
+                                    .mutableCopy());
+                });
     }
 
-    public Command setAngle(IntakePose intakePose) {
+    public Command setAngle(IntakeConstants.IntakePose intakePose) {
         return setAngle(intakePose.intakePose);
     }
 
     public Command setRollerSpeed(double speed) {
-        return runOnce(() -> io.setRollerSpeed(speed));
+        return runOnce(
+                () -> {
+                    io.setRollerSpeed(speed);
+                    inputs.rollerSpeedSetPoint = Units.RotationsPerSecond.of(speed).mutableCopy();
+                });
     }
 
     public Command setCenterRollerSpeed(double speed) {
-        return runOnce(() -> io.setCenterRollerSpeed(speed));
+        return runOnce(
+                () -> {
+                    io.setCenterRollerSpeed(speed);
+                    inputs.centerRollerSpeedSetPoint =
+                            Units.RotationsPerSecond.of(speed).mutableCopy();
+                });
     }
 
     public Command intake() {
-        return setAngle(IntakePose.DOWN)
+        return setAngle(IntakeConstants.IntakePose.DOWN.intakePose)
                 .alongWith(setRollerSpeed(0.5).andThen(setCenterRollerSpeed(0.5)))
                 .withName("feeding position activated");
```

Comment:
The command will indeed work with an `andThen`, but it makes more sense to just run everything in parallel:
```suggestion        
        return Commands.sequence(
                        setAngle(IntakeConstants.IntakePose.DOWN.intakePose),
                        setRollerSpeed(0.5),
                        setCenterRollerSpeed(0.5))
                .withName("feeding position activated");

```
Same applies for the stop command.